Source/ExcelDna.Integration/ExcelDna.Integration.csproj

```
@@ -1,120 +1,21 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

```suggestion
<Project Sdk="Microsoft.NET.Sdk">
```

We should be able to use the regular .NET SDK here (you're already referencing WinForms) down below.

---

I think this is still needed when targeting the framework. It's kind of a bug, and they're fixing for some point version of .NET 5 SDK.